src/lumigo_tracer/wrappers/http/sync_http_wrappers.py

```diff
@@ -36,20 +38,23 @@ def is_lumigo_edge(host: Optional[str]) -> bool:
     return False
 
 
-def add_request_event(parse_params: HttpRequest) -> Dict:
+def add_request_event(span_id: Optional[str], parse_params: HttpRequest) -> Dict:
```

Comment:
Why is it `Optional`? Should it always exist?

Reply:
nope. Sometimes you want to override a span (e.g. when you download a stream, and you get another part from the body), and sometimes you want to create a new span.